grails-app/services/org/pih/warehouse/core/DashboardService.groovy

```diff
@@ -554,6 +554,50 @@ class DashboardService {
         return reorderStock
     }
 
+    def getReorderReport(Location location) {
+        long startTime = System.currentTimeMillis()
+        ArrayList<Map> inventoryItems = getInventoryItems(location)
+
+        ArrayList<Map> reorderStock = inventoryItems.findAll { Map inventoryItem ->
+            def inventoryLevel = inventoryItem?.inventoryLevel as InventoryLevel
+
+            if (inventoryLevel?.binLocation != null) {
+                return false;
+            }
+
+            def quantityATP = inventoryItem.quantityAvailableToPromise as Number
```

Comment:
Nitpick: I used "quantity available to promise" because it's the most precise naming for this value. However, we've standardized on quantityAvailable in most places, so I would continue that trend in order to avoid confusion. We should also consider converting all "quantityAvailableToPromise" references to "quantityAvailable". But we can leave that for now.